style.css

```diff
@@ -1,269 +1,971 @@
+  -webkit-background-clip: text;
+  color: transparent;
+  -webkit-text-fill-color: transparent;
+  text-transform: uppercase;
+}
+
+.main_section_title span {
+  font-size: 6.5rem;
+}
+
+.main_section_description {
+  width: auto;
+  max-width: 350px;
+  margin: 3vh 0 2.5vh 0;
+  font-size: 1.6rem;
+  text-align: left;
+  line-height: 1.5;
+  color: var(--text-color-black);
+  border: 2px solid var(--text-color-light);
+  background-color: #ebf0f1;
+  padding: 1rem;
+}
+
+.main_section_date {
+  font-weight: 900;
+  font-size: 2.3rem;
+}
+
+.main_section_place {
+  font-size: 1.5rem;
+  line-height: 1.25;
+}
+
+/* ------------PROGRAM SECTION----------- */
+.main_program {
+  text-align: center;
+  background-color: var(--main-color-darker);
+  color: #fff;
+  background-image: url(./assets/logos/diamond.png);
+  box-shadow: 0 -3px 5px rgba(0, 0, 0, 0.2);
+}
+
+.section_title {
+  font-size: 2.5rem;
+  font-weight: 900;
+  text-transform: capitalize;
+  padding-top: 5vh;
+}
+
+.div-line {
+  width: 70px;
+  height: 2px;
+  border-radius: 2px;
+  background-color: var(--main-color);
+  margin: 1rem auto;
+}
+
+.main_program_container {
+  width: 96%;
+  max-width: var(--containers-max-width);
+  margin: 2% auto;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-content: center;
+
+  /* gap: 1%; */
+}
+
+.main_program_container_card {
+  flex-basis: 100%;
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: center;
+  gap: 4%;
+  margin: 1rem;
+  padding: 1.5rem;
+  background-color: rgba(50%, 50%, 50%, 0.2);
+  border-radius: 5px;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2);
+  transition: all 0.3s ease;
+}
+
+.main_program_container_card:hover {
+  transform: scale(1.05);
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.3);
+}
+
+.main_program_container_card h3 {
+  flex-basis: 30%;
+  color: var(--main-color);
+  font-size: 1.8rem;
+  font-weight: 900;
+}
+
+.main_program_container_card p {
+  flex-basis: 52%;
+  text-align: left;
+}
+
+.main_program_ext-link {
+  display: none;
+  text-transform: uppercase;
+  text-decoration: underline;
+  color: var(--text-color-white);
+}
+
+.main_program_ext-link h3 {
+  padding-bottom: 5vh;
+}
+
+.main_program_button {
+  width: 80vw;
+  padding: 2rem;
+  margin: 3vh auto 6vh auto;
+  font-size: 2rem;
+  font-weight: 600;
+  letter-spacing: -1px;
+  color: var(--text-color-white);
+  background-color: var(--main-color);
+  border-style: none;
+  cursor: pointer;
+}
+
+.main_program_button:hover {
+  transform: scale(1.05);
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.3);
+}
+
+/* -----------LINEUP SECTION------------ */
+.more-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
+  transition: opacity 0.6s ease;
+}
+
+.see-more {
+  width: 488px;
+  height: 50px;
+  background-color: rgb(255, 255, 255);
+  color: black;
+  border: gray 0.5px solid;
+  padding: 10px 20px;
+  cursor: pointer;
+  transition: opacity 0.6s ease;
+}
```

Comment:
- [ ] Ouch! this is close but your button design doesn't look similar to the original Behance design. First, the width shouldn't take up the entire screen, you can quickly fix that by making the `width: 90%` on line `540`. Second, check the next comment 👇